OpenRA.Game/Server/Server.cs

```diff
@@ -375,7 +375,7 @@ void ValidateClient(Connection newConn, string data)
 				Log.Write("server", "{0} ({1}) has joined the game.",
 					client.Name, newConn.Socket.RemoteEndPoint);
 
-				if (Dedicated || !LobbyInfo.IsSinglePlayer)
+				if (Dedicated || !LobbyInfo.IsSingleHuman)
```

Comment:
This could then simplify down to `if (LobbyInfo.NonBotClients.Count() > 1)` (IMO there's no reason to show the player their own join message on dedicated servers)